Documents/TechnicalSpecifications/technicalSpecifications.md

```diff
@@ -2,32 +2,69 @@
 
 ## Table of content
 
-- [Functional Specifications](#functional-specifications)
+- [Technical specifications](#technical-specifications)
   - [Table of content](#table-of-content)
+  - [Document purpose](#document-purpose)
 - [Project definition](#project-definition)
   - [Overview](#overview)
+  - [Team members](#team-members)
   - [Description](#description)
 - [Development environment](#development-environment)
   - [Hardware](#hardware)
   - [Programming language](#programming-language)
-  - [Conventions](Conventions)
-    - [File tree and documents](#file-tree-and-document)
+  - [Conventions](#conventions)
+    - [File tree and documents](#file-tree-and-documents)
     - [Code](#code)
       - [Naming conventions](#naming-conventions)
       - [Structure conventions](#structure-conventions)
-
+- [Software architecture](#software-architecture)
+  - [Game actions flow](#game-actions-flow)
+  - [Software file management](#software-file-management)
+    - [File tree](#file-tree)
+    - [Files interactions](#files-interactions)
+  - [Program structure](#program-structure)
+- [Risks and mitigations strategy](#risks-and-mitigation-strategies)
+  - [Risks](#risks)
+  - [Mitigation strategies](#mitigation-strategies)
 - [Glossary](#glossary)
 
 
+## Document purpose
+
+The purpose of this document is to describe how the program will be created. It will be updated following the advancements of the project, to match what as been done during the development phase.
```

Comment:
Wrong understanding of what is a technical specifications, this documents purpose is to guide the software engineer not to describe what they did.
You should explain the documents will be updated depending on how the project is going instead of "hat as been done during the development phase."

Reply:
done 👍